src/akkudoktoreos/server/eos.py

```diff
@@ -230,14 +139,43 @@ async def lifespan(app: FastAPI) -> AsyncGenerator[None, None]:
     root_path=str(Path(__file__).parent),
 )
 
-server_dir = Path(__file__).parent.resolve()
-
 
 class PdfResponse(FileResponse):
     media_type = "application/pdf"
 
 
-@app.put("/v1/config/reset", tags=["config"])
+@app.get("/v1/health", status_code=status.HTTP_200_OK)
+async def fastapi_health_get():  # type: ignore
+    """Health check endpoint to verify that the server is alive."""
+    return {"status": "alive"}
+
+
+@app.put("/v1/config/value", tags=["config"])
```

Comment:
Probably merge leftover? /v1/config/value Has to be re-added in the future, currently broken. I would keep /v1/config/reset over .../update to better indicate it's effect.

Reply:
Removed